dags/glam.py

```diff
@@ -366,8 +303,10 @@
 clients_scalar_aggregates >> scalar_percentiles
 clients_scalar_aggregates >> client_scalar_probe_counts
 
-latest_versions >> clients_daily_histogram_aggregates
-clients_daily_histogram_aggregates >> clients_daily_keyed_histogram_aggregates
+latest_versions >> clients_daily_histogram_aggregates_parent
+clients_daily_histogram_aggregates_parent >> clients_daily_histogram_aggregates_content
+clients_daily_histogram_aggregates_parent >> clients_daily_keyed_histogram_aggregates
+clients_daily_histogram_aggregates_content >> clients_histogram_aggregates
```

Comment:
I think we need `clients_daily_keyed_histogram_aggregates >> clients_histogram_aggregates` too here.

Reply:
It's here right below the diff

Reply:
lol oops missed that